pkg/agent/flowexporter/connections/conntrack_windows.go

```diff
@@ -17,11 +17,45 @@
 package connections
 
 import (
+	"fmt"
 	"net"
+	"strconv"
+	"strings"
 
 	"github.com/vmware-tanzu/antrea/pkg/agent/config"
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
 )
 
-func NewConnTrackSystem(nodeConfig *config.NodeConfig, serviceCIDRv4 *net.IPNet, serviceCIDRv6 *net.IPNet, isAntreaProxyEnabled bool) *connTrackOvsCtl {
-	return NewConnTrackOvsAppCtl(nodeConfig, serviceCIDRv4, serviceCIDRv6, isAntreaProxyEnabled)
+type connTrackOvsCtlWindows struct {
+	connTrackOvsCtl
+}
+
+// dpctl/ct-get-maxconns returns operation not supported on Windows node, use dpctl/ct-get-limits intead.
```

Comment:
what's the difference between `dpctl/ct-get-maxconns` and `dpctl/ct-get-limits` in terms of the returned values? I couldn't find an obvious difference by reading the OVS manual

Reply:
According to document, `dpctl/ct-get-maxconns` returns the maximum limit on datapath while `dpctl/ct-get-limits` returns the limit of a zone or zones. I tried to find the value difference between these two commands, however, on Windows node, `dpctl/ct-get-maxconns` command failed and `dpctl/ct-get-limits` command returns 2097152, while on Kind cluster, `dpctl/ct-get-maxconns` returns 300000 and `dpctl/ct-get-limits` command returns 0. So we choose to use `dpctl/ct-get-limits` command only for Windows.